examples/demos/customDateHeader.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import BigCalendar from 'react-big-calendar';
+import events from '../events';
```

Comment:
No need for an example, but please add a story that demonstrates the functionality (in /stories)